src/table_builder.cc

```diff
@@ -30,20 +30,26 @@ void TitanTableBuilder::Add(const Slice& key, const Slice& value) {
 void TitanTableBuilder::AddBlob(const Slice& key, const Slice& value,
                                 std::string* index_value) {
   if (!ok()) return;
+  StopWatch write_sw(db_options_.env, stats_, BLOB_DB_BLOB_FILE_WRITE_MICROS);
 
   if (!blob_builder_) {
     status_ = blob_manager_->NewFile(&blob_handle_);
     if (!ok()) return;
     blob_builder_.reset(
-        new BlobFileBuilder(cf_options_, blob_handle_->GetFile()));
+        new BlobFileBuilder(db_options_, cf_options_, blob_handle_->GetFile()));
   }
 
+  RecordTick(stats_, BLOB_DB_NUM_KEYS_WRITTEN);
+  MeasureTime(stats_, BLOB_DB_KEY_SIZE, key.size());
+  MeasureTime(stats_, BLOB_DB_VALUE_SIZE, value.size());
+
   BlobIndex index;
   BlobRecord record;
   record.key = key;
   record.value = value;
   index.file_number = blob_handle_->GetNumber();
   blob_builder_->Add(record, &index.blob_handle);
+  RecordTick(stats_, BLOB_DB_BLOB_FILE_BYTES_WRITTEN, index.blob_handle.size);
```

Comment:
Move this counter to `BlobFileBuilder`? GC job doesn't use the table builder, only file builder. I want to use this counter to know the total bytes written to blob files.

Reply:
The bytes written by GC is counted in `BLOB_DB_BYTES_WRITTEN`

Reply:
I use `BLOB_DB_BLOB_FILE_BYTES_WRITTEN` to indicate the necessary written bytes to blob file and `BLOB_DB_BYTES_WRITTEN` to indicate the extra written bytes introduced by GC.